docs/user-guide/generate-certificates.md

```diff
@@ -14,32 +40,42 @@ zowe:
   setup:
     certificate:
       type: PKCS12
-      dname:
-        caCommonName: 
-        commonName:
-        orgUnit:
-        org:
-        locality:
-        state:
-        country:
-      validity: 3650
-      pkcs12:
-        directory: /global/zowe/keystore
+        directory: /var/zowe/keystore
+        lock: true
         name: localhost
         password: password
-      caAlias: local_ca
+        caAlias: local_ca
         caPassword: local_ca_password
-        import:
-          keystore:
-          password:
-          alias:
+      dname:
+        caCommonName: ""
+        commonName: ""
+        orgUnit: ""
+        org: ""
+        locality: ""
+        state: ""
+        country: ""
+      validity: 3650
+      san:
+        - dvipa.my-company.com
+        - 12.34.56.78
 ```
 
-To assist with updating `zowe.yaml` the values to generate a self signed PKCS12 certificate are included in the section beginning ` # >>>> Certificate setup scenario 1`.  Other certificate scenarios lower down in the `zowe.yaml` file are commented out.
+To assist with updating `zowe.yaml`, the values to generate a self-signed PKCS12 certificate are included in the section beginning with`# >>>> Certificate setup scenario 1`. Other certificate scenarios lower down in the `zowe.yaml` file are commented out.
+
+### Run the command to generate a PKCS12 certificate
+
+The `zwe init certificate` command generates a certificate based on `zowe.yaml` values in the `zowe.setup.certificate` section. These certificate values used at runtime are referenced in the  `zowe.certificates` section in the `zowe.yaml` file. Specify `--update-config` for the `zwe` command. This setting updates the runtime `zowe.certificates` section to reference the generated certificate generated from the `zowe.setup.certificate`.
```

Comment:
It seems a procedure and example are needed here. 
"Specify `--update-config` for the `zwe` command" is a step. A procedure would seem appropriate.